apps/chronicles/screens/components/CellScreen.tsx

```diff
@@ -76,6 +76,17 @@ export const CellScreen = () => {
                     iconName="navigation-outline"
                     onPress={() => null}
                 />
+                <Cell.Navigation
```

Comment:
Maybe this should go into the `NavigationCellScreen` instead since it is more related to this variant than the generic `Cell` component.